### PR TITLE
Add VPC to k8s manifest (change default to persistent - true)

### PR DESCRIPTION
**Pull request by zamai**
# Pull Request
The plain k8s manifest (not the HELM version), does not have PVC for the kube stateful set. 
This PR adds it for the next developer who wants to grab manifest for working meilisearch without using helm. 

## Related issue
Related issue:, https://github.com/meilisearch/meilisearch-kubernetes/issues/159 

## What does this PR do?
- add's missing PVC to the k8s manifest 

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by zamai** [comment]
@meilisearch sync-manifest

**Comment by zamai** [comment]
@brunoocasali I figured that manifest is generated from helm charts, and I saw that persistence is turned off by default. 

I suggest switching it to `true` by default, so that production scenario (where you defiantly want your search data to persist after pod restart) is the default.

**Comment by brunoocasali** [comment]
Hi @zamai can you update the chart version to v0.4.0? Since you changed a default configuration the chart linters are complaining about that. We can merge it after that!

**Comment by zamai** [comment]
@meilisearch sync-manifest

**Comment by zamai** [comment]
@brunoocasali updated the PR

**Comment by brunoocasali** [comment]
I can't figure out why the CI does not run 😬

**Review by brunoocasali — APPROVED** [review]
LGTM!

bors merge

File: charts/meilisearch/values.yaml [hunk 1]
```diff
@@ -113,7 +113,7 @@ ingress:
 
 persistence:
   # -- Enable persistence using PVC
-  enabled: false
+  enabled: true
```
Comment (brunoocasali) [inline charts/meilisearch/values.yaml:116]: I don't know why I noticed that earlier...

You changed this to `true` but the PR description only mentions that the idea is to provide the PVC for the plain k8s version. Why this have to change then?

File: charts/meilisearch/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v1.5.0"
 description: A Helm chart for the Meilisearch search engine
 name: meilisearch
-version: 0.3.0
+version: 0.4.0
```
Comment (brunoocasali) [inline charts/meilisearch/Chart.yaml:5]: Based on the previous comment, I guess this change could be reverted as well.

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Hey @zamai can you help me with my doubts?

File: charts/meilisearch/values.yaml (hunk 1 above)
Comment (zamai) [reply charts/meilisearch/values.yaml:116]: Initially I thought that plain k8s version is separate from helm charts, but when I saw that sync-manifest command generates k8s yaml files bases on charts values, so I updated this value to be `true`. I think it's a sensible default - having persistent storage for the index of your search cluster.

**Pull request closed without merging**